tests/network/vmi_networking.go

```diff
@@ -257,10 +229,20 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			var err error
 			e1000VMI, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), e1000VMI, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
+			defaultVMI := libvmifact.NewCirros()
+			defaultVMI, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), defaultVMI, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
There is no need to create a second VM, you just need to add a secondary network interface with a default iface mode.

Reply:
Feel free and confirm this and feedback back.

Reply:
Sorry, I do not understand.
How is a different (or similar) domain xml related to the flow?

I'm unsure what is your conclusion here.

Reply:
> The fact that I find the two identical proves to me that I'm testing the same behavior in the test that I've deleted and in the interface that I've added. Hence, interfaces that are connected to primary and secondary networks do not defer in the way that they are emulated to the guest OS.

Comparing the output is similar is at best a hint, not anything beyond.
Code that handles both cases in the same flow is closer to "proof".

Reply:
Yes, we disagree.
Your given argument for similarity is not convincing me, however, because I know the implementation, I know the codebase supports the same logic, be it primary or secondary network.